manage-data/lifecycle/index-lifecycle-management/rollover.md

```diff
@@ -9,50 +9,91 @@ products:
 
-::::{note}
-When an index is rolled over, the previous index’s age is updated to reflect the rollover time. This date, rather than the index’s `creation_date`, is used in {{ilm}} `min_age` phase calculations. [Learn more](../../../troubleshoot/elasticsearch/index-lifecycle-management-errors.md#min-age-calculation).
+* **Size** - an index will rollover when it reaches a set size, for example 50 GB.
+* **Age** - an index will rollover when an index reaches a certain age, for example 7 days.
+* **Document count** - an index will rollover when a shard contains a certain number of documents, for example 2 million.
 
+::::{tip}
+Rolling over to a new index based on size, document count, or age is preferable to time-based rollovers. Rolling over at an arbitrary time often results in many small indices, which can have a negative impact on performance and resource usage.
 ::::
 
+After rollover occurs in the hot phase of an ILM policy, indices move to other index lifecycle phases like warm, cold, frozen, and delete. Rollover creates a new write index while the old one continues through the lifecycle phases.
 
+**Special rules:**
 
-## Automatic rollover [ilm-automatic-rollover]
+* Empty indices rollover is blocked even if they have an associated `max_age` that would otherwise result in a roll over occurring. A policy can override this behavior if you set `min_docs: 0` in the rollover conditions. This can also be disabled on a cluster-wide basis if you set `indices.lifecycle.rollover.only_if_has_documents` to `false`.
+* Forced rollover occurs if any shard reaches 200 million documents. Usually, a shard will reach 50 GB long before it reaches 200 million documents, but this isn’t the case for space efficient data sets.
 
-{{ilm-init}} and the data stream lifecycle enable you to automatically roll over to a new index based on conditions like the index size, document count, or age. When a rollover is triggered, a new index is created, the write alias is updated to point to the new index, and all subsequent updates are written to the new index.
+## Recommended approaches
 
-::::{tip}
-Rolling over to a new index based on size, document count, or age is preferable to time-based rollovers. Rolling over at an arbitrary time often results in many small indices, which can have a negative impact on performance and resource usage.
-::::
+Decide your approach to index rotation based on your use case and requirements.
 
+| Use case               | Recommended approach                                      | Benefits                              |
+| ---------------------- | --------------------------------------------------------- | ------------------------------------- |
+| Logs, metrics          | [Data streams](rollover.md#rollover-data-stream)          | Automatic rollover, minimal setup     |
+| Legacy indexing setup  | [Alias-based rollover](rollover.md#rollover-with-aliases) | Granular control over rollover timing |
```

Comment:
The benefits here are a little unclear: data streams do not automatically have rollover unless they are running in Serverless or using DLM. Likewise, both data streams and aliases with backing indices give control over the rollover timing (for on-prem, in Serverless we do not expose this control).

The biggest difference is in the setup, and how the indices are presented to the user.